docs/snippets/template-v2-scope-and-target.yaml

```diff
@@ -0,0 +1,46 @@
+{% raw %}
+# define your template in a config map
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-config-tpl
+data:
+  config.yaml: |
+    datasources:
+      - name: Graphite
+        type: graphite
+        access: proxy
+        url: http://localhost:8080
+        password: "{{ .password }}"
+        user: "{{ .user }}"
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: my-template-example
+spec:
+  # ...
+  target:
+    name: secret-to-be-created
+    template:
+      engineVersion: v2
+      templateFrom:
+      - scope: Values
+        target: Data
+        configMap:
+          # name of the configmap to pull in
+          name: grafana-config-tpl
+          # here you define the keys that should be used as template
+          items:
+          - key: config.yaml
+      - scope: KeysAndValues
+        target: Annotations
+        literal: "last-sync-for-user/{{ .user }}: {{ .now }}"
```

Comment:
Maybe it is worth to add the example from [this issue](https://github.com/external-secrets/external-secrets/issues/1068#issue-1222327208).

```yaml
  url: {{ .url }}
  {{ if hasPrefix .url "oci://" }}
  enableOCI: "true"
  {{ end }}

```